packages/import-notation/test/parse.test.js

```diff
@@ -70,19 +70,22 @@ describe('block', () => {
         describe('context is block', () => {
             it('should extract blockMod', () => {
                 expect(p('m:autoclosable', { block : 'popup' })).to.eql([
+                    { block : 'popup' },
```

Comment:
it was here on purpose, so it's not a patch 
but since we have a problem with natural dependency in `webpack-bem-loader/babel-plugin-bem-import`: https://github.com/bem/webpack-bem-loader/issues/64
maybe we should fix it here.
Need a discussion. 
And maybe separate pr. Is it possible @Vittly ?

Reply:
Made separate pr: https://github.com/bem/bem-sdk/pull/289

Reply:
hmm, okey I'll do it after adding `expand`